docs/install.md

```diff
@@ -0,0 +1,30 @@
+# Install CVNBot
+
+## Installation
+
+1. Compile the code and build the executable for `Release` configuration (not `Debug`),
+   by running the following command:
+
+   `xbuild src/CVNBot.sln /p:Configuration=Release`
+
+   This command creates files in the output directory at `src/CVNBot/bin/Release`.
+1. Create a directory for your bot, and move the contents of `src/CVNBot/bin/Release` to it.
+1. Edit `CVNBot.ini`: Set at least `botnick`, `botpass` (NickServ login) and `restartarg` (mono + full path to the .exe file)
```

Comment:
In "Release" configuration, `CVNBot.ini`  is not exist in `src/CVNBot/bin/Release`.
I think that should copied from another directory. or.. setting up automatically same as `Debug`?

Reply:
Thanks, I didn't know that it was missing from Release. Yeah, I think the documentation is correct, but the code is wrong. We should copy it automatically, the same as for Debug.

I do not have time this week, but maybe next week. Let's tell each other in IRC when we have time and start the code, so that we don't do the same :)

Reply:
👍

Reply:
See https://github.com/countervandalism/CVNBot/pull/53.